src/sentry/digests/notifications.py

```diff
@@ -200,8 +224,10 @@ def build_digest(project, records, state=None):
 
     state = attach_state(**state)
 
-    def check_group_state(record):
-        return record.value.event.group.get_status() == GroupStatus.UNRESOLVED
+    def check_group_state(record: Record) -> bool:
+        # Explicitly typing to satisfy mypy.
+        is_unresolved: bool = record.value.event.group.get_status() == GroupStatus.UNRESOLVED
```

Comment:
Would annotating `get_status()` solve this?

Reply:
When CI runs mypy it only considers files explicitly included in `mypy.ini`. In order to get `get_status()` annotated, I'd have to annotate `src/sentry/models/group.py` and add it to the list.